listenbrainz/webserver/static/js/src/stats/Bar.test.tsx

```diff
@@ -1,167 +0,0 @@
-import * as React from "react";
-import { shallow } from "enzyme";
-
-import Bar, { BarProps } from "./Bar";
-import * as userRecordingsProcessDataOutput from "../__mocks__/userRecordingsProcessData.json";
-
-const props: BarProps = {
-  data: userRecordingsProcessDataOutput as UserEntityData,
-  maxValue: userRecordingsProcessDataOutput[24].count,
-};
-
-describe("getEntityLink", () => {
```

Comment:
Should we have tests for this function in utils?

Reply:
I have added tests for the function, https://github.com/metabrainz/listenbrainz-server/pull/965/files#diff-03782fa578cb4e20dadcf208fa37c019R5-R15